app/livechat/server/hooks/processRoomAbandonment.js

```diff
@@ -58,5 +58,5 @@ callbacks.add('livechat.closeRoom', (room) => {
 		return;
 	}
 	const secondsSinceLastAgentResponse = getSecondsSinceLastAgentResponse(room, agentLastMessage);
-	LivechatRooms.setVisitorInactivityInSecondsByRoomId(room._id, secondsSinceLastAgentResponse);
+	LivechatRooms.setVisitorInactivityInSecondsById(room._id, secondsSinceLastAgentResponse);
```

Comment:
This method needs to be renamed on the Model as well.
The old name is still there.